ethexe/processor/src/handling/events.rs

```diff
@@ -66,6 +66,8 @@ impl ProcessingHandler {
             return Ok(());
         }
 
+        let dispatch_origin = self.dispatch_origin;
```

Comment:
here it's always from eth, since we literally processing mirror event

Reply:
FIxed